admin/app/controllers/spree/admin/users_controller.rb

```diff
@@ -64,6 +64,10 @@ def collection
                       page(params[:page]).per(params[:per_page])
       end
 
+      def find_resource
+        Spree.user_class.accessible_by(current_ability, :show).find(params[:id])
```

Comment:
There is a `model_class` helper method at line:37.

```suggestion
        model_class.accessible_by(current_ability, :show).find(params[:id])
```